smartling-attachments-api/src/main/java/com/smartling/api/attachments/v2/pto/AttachmentType.java

```diff
@@ -0,0 +1,33 @@
+package com.smartling.api.attachments.v2.pto;
+
+public enum AttachmentType
```

Comment:
where is this enum used?
I only found it in test

Reply:
well... this is kind of enum of supported attachment types just for 'visibility' for those who is going to use sdk.